tests/backends/test_kafka.py

```diff
@@ -132,6 +134,35 @@ def test_auto_offset_reset_latest(self) -> None:
                 else:
                     raise AssertionError("expected EndOfPartition error")
 
+    def test_force_offset_reset_latest(self) -> None:
+        payload = KafkaPayload(b"a", b"0", [])
+        with self.get_topic() as topic:
+            with closing(self.get_producer()) as producer:
+                message = producer.produce(topic, payload).result(5.0)
+
+            # write a nonsense offset
+            with closing(self.get_consumer(auto_offset_reset="error")) as consumer:
```

Comment:
Starting a consumer group with auto_offset_reset="error" is not something we would expect to succeed. Should be "earliest" or "latest".

Reply:
I just copied this from `test_auto_offset_reset_error`. I will change to latest.